src/Extensibility/nunit.extensibility/ExtensionManager.cs

```diff
@@ -124,7 +115,7 @@ public virtual void FindExtensionPoints(params Assembly[] targetAssemblies)
                             string msg = string.Format(
```

Comment:
Did you forgot to convert this one to a formattable string?

Reply:
Done... found a few more as well.